doc/src/programmers-guide.md

```diff
@@ -55,7 +55,8 @@ In addition, several features are supported specifically for integration with mi
 While the list of supported features is long and growing, the currently unsupported Erlang/OTP and BEAM features include (but are not limited to):
 
 * Bignums.  Integer values are restricted to 64-bit values.
-* Bit Syntax.  While packing and unpacking of arbitrary (but less than 64-) bit values is support, packing and unpacking of integer values at the start or end of a binary, or bordering binary packing or extraction must align on 8-bit boundaries.  Arbitrary bit length binaries are not currently supported.
+* Atoms. Atoms cannot be larger than 255 bytes (255 ascii characters) are not supported.
```

Comment:
This is fine, but it might be better to list Atoms first. I think those first entries were intended to be alphabetically listed, and it makes for a smaller change.

Reply:
Done - should be in the correct order now. Can't believe I didn't catch the ordering.